src/app/donation-start/donation-start-form/donation-start-form.component.ts

```diff
@@ -741,7 +807,12 @@ export class DonationStartFormComponent implements AfterContentChecked, AfterCon
   }
 
   async submit() {
-    if (this.donation && this.donationForm.invalid) {
+    // Can't proceed if donation or campaign data issue. Prior check fn has already reported the issue.
+    if (this.donationUpdateError) {
+      return;
```

Comment:
the user just clicked the button now though didn't they? I think we should show an error toast any time the user clicks and we can't do what they want, even if we have already told them about the issue. The alternative is to disable the button but we moved away from that.

Reply:
So I guess if the user goes to the confirm step, `donationUpdateError` is set true because `update` throws. 

Then the user takes as long as they like to think about it, forgets what the issue was, and then they click `Donate now` which is directly bound to this submit function.

Just thought we should repeat the toast to remind them.

Reply:
Thanks, as discussed I was confused about the specific event. Donors could have entered the step seconds or minutes before so we should show an extra toast if the button is to remain active. Done in 00d2a9dc0fdbfed2d474fcb54977eb66d915fbf3